plugins/nodebb-plugin-caiz/libs/x-notification/x-notification.js

```
@@ -0,0 +1,110 @@
+        cid = data.topic ? data.topic.cid : data.cid;
+        break;
+      case 'newPost':
+        if (data.post) {
+          const topicData = await topics.getTopicData(data.post.tid);
+          cid = topicData.cid;
+        }
+        break;
+      case 'memberJoin':
+      case 'memberLeave':
+        // Extract community ID from group name (e.g., "cid:1:members")
+        const match = data.groupName.match(/^cid:(\d+):members$/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)